models/user/user.go

```diff
@@ -41,6 +41,9 @@ const (
 
 	// UserTypeOrganization defines an organization
 	UserTypeOrganization
+
+	// UserTypeReserved defines a reserved user, which is used for internal purposes
```

Comment:
```suggestion
	// UserTypeReserved reserves a (non-existing) user,  i.e. to prevent a spam user from re-registering after being deleted, or to reserve the name until the user is actually created later on
```